index.html

```diff
@@ -2359,7 +2359,7 @@ <h5>Character naming template</h5>
 
 <p>The <code translate="no" class="kw">bdi</code> element is used to ensure that example characters that are right-to-left do not interfere with the layout of the page. Do not include line breaks or a space between the closing <code translate="no" class="kw">bdi</code> and the following <code translate="no" class="kw">span</code> element; spacing and presentation is controlled by styling.</p>
 
-<p>The <code translate="no" class="kw">lang</code> attribute should be filled in appropriately to get the correct font selection for a given context. Examples in East Asian languages (such as Chinese, Japanese, or Korean) or in the Arabic script can sometimes require greater care in choosing a language tag. In some cases, it might also be necessary for you to adjust the <kbd translate="no">font-family</kbd> and/or <kbd translate="no">font-size</kbd> in your own style sheet for characters or character sequences in a given language.</p>
+<p>The <code translate="no" class="kw">lang</code> attribute should be filled in appropriately to get the correct font selection for a given context. Examples in East Asian languages (such as Chinese, Japanese, or Korean) or in the Arabic script can sometimes require greater care in choosing a language tag. Rarely, for certain languages, it might be necessary to adjust the style of the <code>bdi</code> element with a <kbd translate="no">font-family</kbd> and/or <kbd translate="no">font-size</kbd> in your own stylesheet.</p>
```

Comment:
I'm inclined to think that it's not nessecelery rare, most non-latin usages will probably need some styling.

Reply:
Most users will be dealing with ASCII and perhaps some specific characters, so this "rarely" is targeted at them (rather than at us).